sgkit/io/vcf/vcf_writer.py

```diff
@@ -25,16 +28,59 @@
 )
 from sgkit.typing import PathType
 
+# [Table 1: Reserved INFO keys]
```

Comment:
Good to note what version of VCF this corresponds to, it'll make it easier to stay up to date over time.